Description: Fix broken code
 nkf.1j file contains some invaild Japanese string.
 It was fixed on upstream git c778b37.
 .
 nkf (2.13-2) unstable; urgency=low
 .
   * Changed debian/rules format to dh7.
   * Import nkf.1j from git c778b37, closes: #752736.
     http://sourceforge.jp/ticket/browse.php?group_id=248&tid=32546
Author: NOKUBI Takatsugu <knok@daionet.gr.jp>
Bug-Debian: http://bugs.debian.org/752736

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: upstream, http://sourceforge.jp/projects/nkf/scm/git/nkf/commits/c778b37ecf951d50195768deea665a3ebc9f9af6
Bug: http://sourceforge.jp/ticket/browse.php?group_id=248&tid=32546
Bug-Debian: http://bugs.debian.org/752736
Forwarded: not-needed
Last-Update: 2014-06-26

--- nkf-2.13.orig/nkf.1j
+++ nkf-2.13/nkf.1j
@@ -59,68 +59,6 @@
 .    de IX
 ..
 .\}
-.\"
-.\" Accent mark definitions (@(#)ms.acc 1.5 88/02/08 SMI; from UCB 4.2).
-.\" Fear.  Run.  Save yourself.  No user-serviceable parts.
-.    \" fudge factors for nroff and troff
-.if n \{\
-.    ds #H 0
-.    ds #V .8m
-.    ds #F .3m
-.    ds #[ \f1
-.    ds #] \fP
-.\}
-.if t \{\
-.    ds #H ((1u-(\\\\n(.fu%2u))*.13m)
-.    ds #V .6m
-.    ds #F 0
-.    ds #[ \&
-.    ds #] \&
-.\}
-.    \" simple accents for nroff and troff
-.if n \{\
-.    ds ' \&
-.    ds ` \&
-.    ds ^ \&
-.    ds , \&
-.    ds ~ ~
-.    ds /
-.\}
-.if t \{\
-.    ds ' \\k:\h'-(\\n(.wu*8/10-\*(#H)'\'\h"|\\n:u"
-.    ds ` \\k:\h'-(\\n(.wu*8/10-\*(#H)'\`\h'|\\n:u'
-.    ds ^ \\k:\h'-(\\n(.wu*10/11-\*(#H)'^\h'|\\n:u'
-.    ds , \\k:\h'-(\\n(.wu*8/10)',\h'|\\n:u'
-.    ds ~ \\k:\h'-(\\n(.wu-\*(#H-.1m)'~\h'|\\n:u'
-.    ds / \\k:\h'-(\\n(.wu*8/10-\*(#H)'\z\(sl\h'|\\n:u'
-.\}
-.    \" troff and (daisy-wheel) nroff accents
-.ds : \\k:\h'-(\\n(.wu*8/10-\*(#H+.1m+\*(#F)'\v'-\*(#V'\z.\h'.2m+\*(#F'.\h'|\\n:u'\v'\*(#V'
-.ds 8 \h'\*(#H'\(*b\h'-\*(#H'
-.ds o \\k:\h'-(\\n(.wu+\w'\(de'u-\*(#H)/2u'\v'-.3n'\*(#[\z\(de\v'.3n'\h'|\\n:u'\*(#]
-.ds d- \h'\*(#H'\(pd\h'-\w'~'u'\v'-.25m'\f2\(hy\fP\v'.25m'\h'-\*(#H'
-.ds D- D\\k:\h'-\w'D'u'\v'-.11m'\z\(hy\v'.11m'\h'|\\n:u'
-.ds th \*(#[\v'.3m'\s+1I\s-1\v'-.3m'\h'-(\w'I'u*2/3)'\s-1o\s+1\*(#]
-.ds Th \*(#[\s+2I\s-2\h'-\w'I'u*3/5'\v'-.3m'o\v'.3m'\*(#]
-.ds ae a\h'-(\w'a'u*4/10)'e
-.ds Ae A\h'-(\w'A'u*4/10)'E
-.    \" corrections for vroff
-.if v .ds ~ \\k:\h'-(\\n(.wu*9/10-\*(#H)'\s-2\u~\d\s+2\h'|\\n:u'
-.if v .ds ^ \\k:\h'-(\\n(.wu*10/11-\*(#H)'\v'-.4m'^\v'.4m'\h'|\\n:u'
-.    \" for low resolution devices (crt and lpr)
-.if \n(.H>23 .if \n(.V>19 \
-\{\
-.    ds : e
-.    ds 8 ss
-.    ds o a
-.    ds d- d\h'-1'\(ga
-.    ds D- D\h'-1'\(hy
-.    ds th \o'bp'
-.    ds Th \o'LP'
-.    ds ae ae
-.    ds Ae AE
-.\}
-.rm #[ #] #H #V #F C
 .\" ========================================================================
 .\"
 .IX Title "nkf 1"
@@ -130,251 +68,251 @@
 .if n .ad l
 .nh
 .SH "NAME"
-nkf \- XXXXXXXXXXXXXXXXXX
+nkf \- ネットワーク用漢字コード変換フィルタ
 .SH "SYNOPSIS"
 .IX Header "SYNOPSIS"
 nkf \fB[\-butjnesliohrTVvwWJESZxXFfmMBOcdILg]\fR \fB[\fR\fIfile ...\fR\fB]\fR
 .SH "DESCRIPTION"
 .IX Header "DESCRIPTION"
-\&\fBnkf\fR \s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
+\&\fBnkf\fR はネットワークでメールやニュースの読み書きをするために作られた、漢字コードの変換フィルタである。
 .PP
-\&\s-1XX\s0\fBnkf\fR\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
+この\fBnkf\fRの特徴としては、入力漢字コード系の統計的な自動認識機能がある。
+このため、利用者は、入力漢字コード系が何であるかを知らなくても、出力漢字コード系のみ指定すれば良いことになる。
+ただ、この判定機構は、理論的には完全ではないが、通常のニュースやメールのメッセージについては確実に動作する安全なものにはなっている。
 .PP
-\&\s-1XXX\s0\fBnkf\fR\s-1XXXXXXXXXXXXXXXXXXXXX\s0 \s-1JIS\s0 \s-1XXX\s0 (\s-1ISO\-2022\-JP\s0 \s-1XXXXXX\s0)XShift_JIS (\s-1MS\s0 \s-1XXXXX\s0)X
-\&\s-1XXX\s0 \s-1EUC\s0 (\s-1AT&T\s0 \s-1XXX\s0)XUTF\-8XUTF\-16 \s-1XXXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXX\s0
+現在、\fBnkf\fRが認識できる入力の漢字コード系は、いわゆる \s-1JIS\s0 コード (\s-1ISO\-2022\-JP\s0 に基づくもの)、Shift_JIS (\s-1MS\s0 漢字コード)、
+日本語 \s-1EUC\s0 (\s-1AT&T\s0 コード)、UTF\-8、UTF\-16 のいずれかである。
+出力する漢字コード系も、これらである。
+入力は、ファイルを指定しなければ、標準入力となる。
+出力は標準出力である。
 .SH "OPTIONS"
 .IX Header "OPTIONS"
-\&\s-1XXXXXXXXXXXXXXXXXX\s0
-\&\fB\-mu\fR \s-1XXXXXXXXXXXXXX\s0
+指定できるオプションは、以下の通り。
+\&\fB\-mu\fR のように続けることができる。
 .IP "\fB\-j\fR" 4
 .IX Item "-j"
-\&\s-1JIS\s0 \s-1XXXXXXXXX\s0
+\&\s-1JIS\s0 コードを出力する。
 .IP "\fB\-e\fR" 4
 .IX Item "-e"
-\&\s-1EUC\s0 \s-1XXXXXXXXX\s0
+\&\s-1EUC\s0 コードを出力する。
 .IP "\fB\-s\fR" 4
 .IX Item "-s"
-Shift_JIS \s-1XXXXXXXXX\s0
+Shift_JIS コードを出力する。
 .IP "\fB\-w \-w8[0] \-w16[\s-1BL\s0][0]\fR" 4
 .IX Item "-w -w8[0] -w16[BL][0]"
-Unicode \s-1XXXXXX\s0
+Unicode を出力する。
 .RS 4
 .IP "\fB\-w \-w80\fR" 4
 .IX Item "-w -w80"
-\&\s-1UTF8\s0 \s-1XXXXXXXXX\s0 (\s-1BOM\s0 \s-1XX\s0)
+\&\s-1UTF8\s0 コードを出力する。 (\s-1BOM\s0 無し)
 .IP "\fB\-w8\fR" 4
 .IX Item "-w8"
-\&\s-1UTF8\s0 \s-1XXXXXXXXX\s0
+\&\s-1UTF8\s0 コードを出力する。
 .IP "\fB\-w16 \-w16B0\fR" 4
 .IX Item "-w16 -w16B0"
-\&\s-1UTF16\s0 \s-1XXXXXXXXX\s0 (Big Endian / \s-1BOM\s0 \s-1XX\s0)
+\&\s-1UTF16\s0 コードを出力する。 (Big Endian / \s-1BOM\s0 無し)
 .IP "\fB\-w16B\fR" 4
 .IX Item "-w16B"
-\&\s-1UTF16\s0 \s-1XXXXXXXXX\s0 (Big Endian / \s-1BOM\s0 \s-1XX\s0)
+\&\s-1UTF16\s0 コードを出力する。 (Big Endian / \s-1BOM\s0 有り)
 .IP "\fB\-w16L\fR" 4
 .IX Item "-w16L"
-\&\s-1UTF16\s0 \s-1XXXXXXXXX\s0 (Little Endian / \s-1BOM\s0 \s-1XX\s0)
+\&\s-1UTF16\s0 コードを出力する。 (Little Endian / \s-1BOM\s0 有り)
 .IP "\fB\-w16L0\fR" 4
 .IX Item "-w16L0"
-\&\s-1UTF16\s0 \s-1XXXXXXXXX\s0 (Little Endian / \s-1BOM\s0 \s-1XX\s0)
+\&\s-1UTF16\s0 コードを出力する。 (Little Endian / \s-1BOM\s0 無し)
 .RE
 .RS 4
 .RE
 .IP "\fB\-m[\s-1BQSN0\s0]\fR" 4
 .IX Item "-m[BQSN0]"
-\&\s-1MIME\s0 \s-1XXXXXX\s0(\s-1XXXXX\s0)
-\&\s-1ISO\-2022\-JP\s0 (B encode) X \s-1ISO\-8859\-1\s0 (Q encode) \s-1XXXXXXXX\s0
-\&\s-1ISO\-8859\-1\s0 (Latin\-1) \s-1XXXXXXXX\s0\fB\-l\fR \s-1XXXXXXXXXX\s0
-\&\fB\-m0\fR \s-1XX\s0 \s-1MIME\s0 \s-1XXXXXXX\s0
-\&\fB\-mB\fR, \fB\-mQ\fR \s-1XXXBASE64\s0, Q encode \s-1XXXXXXXXXXXXXXX\s0
+\&\s-1MIME\s0 を解読する。(デフォルト)
+\&\s-1ISO\-2022\-JP\s0 (B encode) と \s-1ISO\-8859\-1\s0 (Q encode) のみを解読する。
+\&\s-1ISO\-8859\-1\s0 (Latin\-1) を解読する時は、\fB\-l\fR フラグも必要である。
+\&\fB\-m0\fR では \s-1MIME\s0 を解読しない。
+\&\fB\-mB\fR, \fB\-mQ\fR では、BASE64, Q encode されているものとして処理する。
 .RS 4
 .IP "\fB\-mB\fR" 4
 .IX Item "-mB"
-\&\s-1MIME\s0 base64 stream \s-1XXXXXX\s0
-\&\s-1XXXXXXXXXXXXX\s0
+\&\s-1MIME\s0 base64 stream を解読する。
+ヘッダなどは取り除くこと。
 .IP "\fB\-mQ\fR" 4
 .IX Item "-mQ"
-\&\s-1MIME\s0 quoted stream \s-1XXXXXX\s0
+\&\s-1MIME\s0 quoted stream を解読する。
 .IP "\fB\-mS\fR" 4
 .IX Item "-mS"
-\&\s-1MIME\s0 \s-1XXXXXXXXXXX\s0 (\s-1XXXXX\s0)
+\&\s-1MIME\s0 のチェックを厳しくする (デフォルト)
 .IP "\fB\-mN\fR" 4
 .IX Item "-mN"
-\&\s-1MIME\s0 \s-1XXXXXXXXXX\s0
+\&\s-1MIME\s0 のチェックを緩くする
 .IP "\fB\-m0\fR" 4
 .IX Item "-m0"
-\&\s-1MIME\s0 \s-1XXXXXXX\s0
+\&\s-1MIME\s0 を解読しない。
 .RE
 .RS 4
 .Sp
-\&\fB\-mS\fR \s-1XXXXXXXXXXXXXXX\s0 encoded-text \s-1XXXXXXXXXXXX\s0
-\&\fB\-mN\fR \s-1XXXXXXXXX\s0 \s-1MIME\s0 \s-1XXXXXXXXXXXXXXXXXXXXXX\s0
+\&\fB\-mS\fR は、宣言の内容と、その後に続く encoded-text の整合性をチェックする。
+\&\fB\-mN\fR は、改行で切られた \s-1MIME\s0 なども解読する。解読が止まらないこともある。
 .RE
 .IP "\fB\-M[\s-1BQ\s0]\fR" 4
 .IX Item "-M[BQ]"
-\&\s-1MIME\s0 \s-1XXXXXX\s0
+\&\s-1MIME\s0 に変換する。
 .RS 4
 .IP "\fB\-M\fR" 4
 .IX Item "-M"
-\&\s-1XXXXXXXXXX\s0
+ヘッダ形式に変換する
 .IP "\fB\-MB\fR" 4
 .IX Item "-MB"
-base64 stream \s-1XXXXX\s0
+base64 stream に変換する
 .IP "\fB\-MQ\fR" 4
 .IX Item "-MQ"
-Quoted stream \s-1XXXXX\s0
+Quoted stream に変換する
 .RE
 .RS 4
 .RE
 .IP "\fB\-J \-E \-S \-W\fR" 4
 .IX Item "-J -E -S -W"
-\&\s-1XXXXXXXXXXXXXXXXXXX\s0
+期待される入力コードの性質を指定する。
 .RS 4
 .IP "\fB\-J\fR" 4
 .IX Item "-J"
-\&\s-1ISO\-2022\-JP\s0 \s-1XXXXXX\s0
+\&\s-1ISO\-2022\-JP\s0 を仮定する。
 .IP "\fB\-E\fR" 4
 .IX Item "-E"
-\&\s-1XXX\s0 \s-1EUC\s0 (\s-1AT&T\s0) \s-1XXXXXX\s0
+日本語 \s-1EUC\s0 (\s-1AT&T\s0) を仮定する。
 .IP "\fB\-S\fR" 4
 .IX Item "-S"
-Shift_JIS \s-1XXXXXX\s0
-\&\s-1XXXXXXXX\s0 (\s-1JIS\s0 X 0201 \s-1XXX\s0) \s-1XXXXXXX\s0
+Shift_JIS を仮定する。
+いわゆる半角カナ (\s-1JIS\s0 X 0201 片仮名) も受け入れる。
 .IP "\fB\-W\fR" 4
 .IX Item "-W"
-\&\s-1UTF\-8\s0 \s-1XXXXXX\s0
+\&\s-1UTF\-8\s0 を仮定する。
 .IP "\fB\-W8\fR" 4
 .IX Item "-W8"
-\&\s-1UTF\-8\s0 \s-1XXXXXX\s0
+\&\s-1UTF\-8\s0 を仮定する。
 .IP "\fB\-W16\fR" 4
 .IX Item "-W16"
-\&\s-1UTF\-16\s0 (Little Endian)XXXXXX
+\&\s-1UTF\-16\s0 (Little Endian)を仮定する。
 .IP "\fB\-W16B\fR" 4
 .IX Item "-W16B"
-\&\s-1UTF\-16\s0 (Big Endian)XXXXXX
+\&\s-1UTF\-16\s0 (Big Endian)を仮定する。
 .IP "\fB\-W16L\fR" 4
 .IX Item "-W16L"
-\&\s-1UTF\-16\s0 (Little Endian)XXXXXX
+\&\s-1UTF\-16\s0 (Little Endian)を仮定する。
 .RE
 .RS 4
 .RE
 .IP "\fB\-x\fR" 4
 .IX Item "-x"
-\&\s-1XXXXXXXXXXXXXXXXX\s0 (\s-1JIS\s0 X 0201 \s-1XXX\s0)
-\&\s-1XXXXXXXXXX\s0 (\s-1JIS\s0 X 0208 \s-1XXX\s0) \s-1XXXXXXXXXXXXXXXXXXX\s0
-XXXXShift_JIS X 1byte \s-1XXXSO/SIXESC\-\s0(\-I, \s-1SSO\s0 \s-1XXXXXXX\s0
-\&\s-1XXXXXXX\s0 \s-1EUC\s0 \s-1XXX\s0 \s-1SSOXJIS\s0 \s-1XXXXXX\s0 \s-1ESC\-\s0(\-I \s-1XXXXXXXXXXX\s0
+通常おこなわれる、いわゆる半角カナ (\s-1JIS\s0 X 0201 片仮名)
+からいわゆる全角カナ (\s-1JIS\s0 X 0208 片仮名) への変換を行わず、半角カナを保存する。
+入力は、Shift_JIS の 1byte カナ、SO/SI、ESC\-(\-I, \s-1SSO\s0 を受け付ける。
+出力は、日本語 \s-1EUC\s0 中では SSO、JIS コード中では \s-1ESC\-\s0(\-I をデフォルトで用いる。
 .IP "\fB\-X\fR" 4
 .IX Item "-X"
-\&\s-1XXXXXXXX\s0 (\s-1JIS\s0 X 0201 \s-1XXX\s0) X \s-1XXXXXXXX\s0 (\s-1JIS\s0 X 0208 \s-1XXX\s0) \s-1XXXXXXX\s0
+いわゆる半角カナ (\s-1JIS\s0 X 0201 片仮名) を いわゆる全角カナ (\s-1JIS\s0 X 0208 片仮名) へと変換する。
 .IP "\fB\-B\fR" 4
 .IX Item "-B"
-\&\s-1XXX\s0 (Broken) \s-1JIS\s0 \s-1XXXX\s0
-\&\s-1ESC\s0 \s-1XXXXXXXXXXXX\s0
+壊れた (Broken) \s-1JIS\s0 コード。
+\&\s-1ESC\s0 がなくなったと仮定する。
 .RS 4
 .IP "\fB\-B1\fR" 4
 .IX Item "-B1"
-\&\fB\s-1ESC\-\s0(\fR, \fB\s-1ESC\-$\s0\fR \s-1XXXXXXXXXXXX\s0
+\&\fB\s-1ESC\-\s0(\fR, \fB\s-1ESC\-$\s0\fR のあとのコードを問わない
 .IP "\fB\-B2\fR" 4
 .IX Item "-B2"
-\&\s-1XXXXXXXXXX\s0 \s-1ASCII\s0 \s-1XXX\s0
+改行のあとに強制的に \s-1ASCII\s0 に戻す
 .RE
 .RS 4
 .RE
 .IP "\fB\-f\fR[\fIm\fR[\-\fIn\fR]]" 4
 .IX Item "-f[m[-n]]"
-\&\s-1XX\s0 \fIm\fR \s-1XXXXXXXXXXXXXX\s0 \fIn\fR \s-1XXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXX\s0 60XXXXXX 10 \s-1XXXX\s0
+一行 \fIm\fR 文字になるように、マージンを \fIn\fR として簡単な整形をおこなう。
+デフォルトは文字数は 60、マージンは 10 である。
 .IP "\fB\-Z[0\-3]\fR" 4
 .IX Item "-Z[0-3]"
-\&\s-1JIS\s0 X 0208 \s-1XXXXXXXXXX\s0 \s-1ASCII\s0 \s-1XXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXX\s0
+\&\s-1JIS\s0 X 0208 英数字と若干の記号を \s-1ASCII\s0 に変換する。
+つまり、全角を半角に変換する。
 .RS 4
 .IP "\fB\-Z \-Z0\fR" 4
 .IX Item "-Z -Z0"
 Convert X0208 alphabet to \s-1ASCII\s0.
 .IP "\fB\-Z1\fR" 4
 .IX Item "-Z1"
-\&\s-1JIS\s0 X 0208 \s-1XXXXX\s0 \s-1ASCII\s0 space \s-1XXXXXXXX\s0
+\&\s-1JIS\s0 X 0208 和字間隔を \s-1ASCII\s0 space 一つに変換する。
 .IP "\fB\-Z2\fR" 4
 .IX Item "-Z2"
-\&\s-1JIS\s0 X 0208 \s-1XXXXX\s0 \s-1ASCII\s0 space \s-1XXXXXXXX\s0
+\&\s-1JIS\s0 X 0208 和字間隔を \s-1ASCII\s0 space 二つに変換する。
 .IP "\fB\-Z3\fR" 4
 .IX Item "-Z3"
-\&\s-1XXXXXXXXX\s0 &gt;X&lt;X&quot;X&amp; \s-1XXXXXX\s0
+＞、＜、”、＆、を &gt;、&lt;、&quot;、&amp; に変換する。
 .RE
 .RS 4
 .RE
 .IP "\fB\-b\fR" 4
 .IX Item "-b"
-\&\s-1XXXXXXXXXXXXX\s0(\s-1XXXXX\s0)
+バッファリング出力を行う。(デフォルト)
 .IP "\fB\-u\fR" 4
 .IX Item "-u"
-\&\s-1XXXXXXXXXXXXXXXX\s0
-ssh localhost | nkf \-u \s-1XXXXXXXXX\s0
+出力時に、バッファリングしない。
+ssh localhost | nkf \-u というように使う。
 .IP "\fB\-t\fR" 4
 .IX Item "-t"
-\&\s-1XXXXXX\s0
+何もしない。
 .IP "\fB\-I\fR" 4
 .IX Item "-I"
-\&\s-1ISO\-2022\-JP\s0 \s-1XXXXXXXXXXXXXX\s0
+\&\s-1ISO\-2022\-JP\s0 以外の漢字コードを〓に変換。
 .IP "\fB\-i[@B]\fR" 4
 .IX Item "-i[@B]"
-\&\s-1JIS\s0 \s-1XXXXXXXXXXXXXXXXXX\s0
+\&\s-1JIS\s0 漢字を指示するシーケンスを指定する。
 .RS 4
 .IP "\fB\-i@\fR" 4
 .IX Item "-i@"
-\&\s-1JIS\s0 X 0208\-1978 \s-1XXXXX\s0
+\&\s-1JIS\s0 X 0208\-1978 を用いる。
 .IP "\fB\-iB\fR" 4
 .IX Item "-iB"
-\&\s-1JIS\s0 X 0208\-1983/1990 \s-1XXXXX\s0
+\&\s-1JIS\s0 X 0208\-1983/1990 を用いる。
 .RE
 .RS 4
 .RE
 .IP "\fB\-o[\s-1BJH\s0]\fR" 4
 .IX Item "-o[BJH]"
-1 \s-1XXXXXXXXXXXXXXXXXXXXXXXXXX\s0
+1 バイト英数文字セットを指示するシーケンスを指定する。
 .RS 4
 .IP "\fB\-oB\fR" 4
 .IX Item "-oB"
-1 \s-1XXXXXXXXXXXXX\s0 US-ASCII \s-1XXXXX\s0
+1 バイト英数文字セットとして US-ASCII を用いる。
 .IP "\fB\-oJ\fR" 4
 .IX Item "-oJ"
-1 \s-1XXXXXXXXXXXXX\s0 \s-1JIS\s0 X 0201 Roman \s-1XXXXX\s0
+1 バイト英数文字セットとして \s-1JIS\s0 X 0201 Roman を用いる。
 .IP "\fB\-oH\fR" 4
 .IX Item "-oH"
-\&\s-1ESC\s0 ( H \s-1XXXX\s0 \s-1JUNET\s0 \s-1XXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
+\&\s-1ESC\s0 ( H は初期の \s-1JUNET\s0 コードでのみ用いられたもので、現在は使ってはならない。
 .RE
 .RS 4
 .RE
 .IP "\fB\-r\fR" 4
 .IX Item "-r"
-\&\s-1ROT13/47\s0 \s-1XXXXXXX\s0
+\&\s-1ROT13/47\s0 の変換をする。
 .IP "\fB\-g\fR" 4
 .IX Item "-g"
-\&\s-1XXXXXXXXXXXXXX\s0
+自動判別の結果を出力します。
 .IP "\fB\-T\fR" 4
 .IX Item "-T"
-\&\s-1XXXXXXXXXXXXX\s0(\s-1MS\-DOS\s0 \s-1XXXXXXXXX\s0)
+テキストモードで出力する。(\s-1MS\-DOS\s0 上でのみ効力を持つ)
 .IP "\fB\-l\fR" 4
 .IX Item "-l"
-0x80\-0xfe \s-1XXXXX\s0 \s-1ISO\-8859\-1\s0 (Latin\-1) \s-1XXXXXX\s0
-\&\s-1JIS\s0 \s-1XXXXXXXXXXXXXXXXXXXX\s0
-\&\fB\-s, \-e, \-x\fR \s-1XXXXXXXX\s0
+0x80\-0xfe のコードを \s-1ISO\-8859\-1\s0 (Latin\-1) として扱う。
+\&\s-1JIS\s0 コードアウトプットとの組合せみのみ有効。
+\&\fB\-s, \-e, \-x\fR とは両立しない。
 .IP "\fB\-O\fR" 4
 .IX Item "-O"
-\&\s-1XXXXXXXXXX\s0
-\&\s-1UNIX\s0 \s-1XXXXXXXXX\s0 Windows X \s-1MSDOS\s0 \s-1XXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXX\s0 nkf.outX
+ファイルに出力する。
+\&\s-1UNIX\s0 では不要な機能だが Windows や \s-1MSDOS\s0 では必要らしい。
+直後の引き数でなく、最後のファイル名が出力ファイル名となり上書きされてしまうので注意。
+ファイル名がない場合は nkf.out。
 .IP "\fB\-L[uwm] \-d \-c\fR" 4
 .IX Item "-L[uwm] -d -c"
-\&\s-1XXXXXXXXXXX\s0
+改行コードを変換する。
 .RS 4
 .IP "\fB\-Lu \-d\fR" 4
 .IX Item "-Lu -d"
@@ -388,171 +326,171 @@ mac (\s-1CR\s0)
 .RE
 .RS 4
 .Sp
-\&\s-1XXXXXXXXXXXXX\s0
+デフォルトでは変換しない。
 .RE
 .IP "\fB\-\-ic=<input_codeset\fR \-\-oc=<output_codeset>>" 4
 .IX Item "--ic=<input_codeset --oc=<output_codeset>>"
-\&\s-1XXXXXXXXXXXXXXXXXXX\s0
+入力・出力の漢字コード系を指定します。
 .RS 4
 .IP "\fB\s-1ISO\-2022\-JP\s0\fR" 4
 .IX Item "ISO-2022-JP"
-\&\s-1XXXX\s0 \s-1JIS\s0 \s-1XXXX\s0\fB\-j, \-J\fR \s-1XXXX\s0
+いわゆる \s-1JIS\s0 コード。\fB\-j, \-J\fR と同じ。
 .IP "\fB\s-1ISO\-2022\-JP\-1\s0\fR" 4
 .IX Item "ISO-2022-JP-1"
-\&\s-1RFC\s0 2237 \s-1XXXXXXXXX\s0 \s-1JIS\s0 X 0212 \s-1XXXX\s0
+\&\s-1RFC\s0 2237 に定められた形式。 \s-1JIS\s0 X 0212 を含む。
 .IP "\fB\s-1ISO\-2022\-JP\-3\s0\fR" 4
 .IX Item "ISO-2022-JP-3"
-\&\s-1RFC\s0 2237 \s-1XXXXXXXXX\s0 \s-1JIS\s0 X 0213 \s-1XXXX\s0
+\&\s-1RFC\s0 2237 に定められた形式。 \s-1JIS\s0 X 0213 を含む。
 .IP "\fBEUC-JP\fR" 4
 .IX Item "EUC-JP"
-\&\s-1EUC\s0 \s-1XXXX\s0\fB\-e, \-E\fR \s-1XXXX\s0
+\&\s-1EUC\s0 コード。\fB\-e, \-E\fR と同じ。
 .IP "\fB\s-1EUC\-JISX0213\s0\fR" 4
 .IX Item "EUC-JISX0213"
-\&\s-1XXXXX\s0 \s-1JIS\s0 X 0213:2000 \s-1XXXX\s0 EUC-JPX
+文字集合に \s-1JIS\s0 X 0213:2000 を用いた EUC\-JP。
 .IP "\fB\s-1EUC\-JIS\-2004\s0\fR" 4
 .IX Item "EUC-JIS-2004"
-\&\s-1XXXXX\s0 \s-1JIS\s0 X 0213:2004 \s-1XXXX\s0 EUC-JPX
+文字集合に \s-1JIS\s0 X 0213:2004 を用いた EUC\-JP。
 .IP "\fBeucJP-ascii\fR" 4
 .IX Item "eucJP-ascii"
-\&\s-1XXXXXXXXXXXXXXXXXXXXX\s0 eucJP-asciiX
-\&\fB\-x\fR \s-1XXXXXXXXXXXXX\s0
+オープングループ日本ベンダ協議会が定義した eucJP\-ascii。
+\&\fB\-x\fR が暗黙のうちに指定される。
 .IP "\fBeucJP-ms\fR" 4
 .IX Item "eucJP-ms"
-\&\s-1XXXXXXXXXXXXXXXXXXXXX\s0 euc-JPmsX
-\&\fB\-x\fR \s-1XXXXXXXXXXXXX\s0
+オープングループ日本ベンダ協議会が定義した euc\-JPms。
+\&\fB\-x\fR が暗黙のうちに指定される。
 .IP "\fB\s-1CP51932\s0\fR" 4
 .IX Item "CP51932"
-Micorosft Code Page 51932X
-\&\fB\-x\fR \s-1XXXXXXXXXXXXX\s0
+Micorosft Code Page 51932。
+\&\fB\-x\fR が暗黙のうちに指定される。
 .IP "\fBShift_JIS\fR" 4
 .IX Item "Shift_JIS"
-Shift_JISX
-\&\fB\-s, \-S\fR \s-1XXXX\s0
+Shift_JIS。
+\&\fB\-s, \-S\fR と同じ。
 .IP "\fBShift_JISX0213\fR" 4
 .IX Item "Shift_JISX0213"
-\&\s-1XXXXX\s0 \s-1JIS\s0 X 0213:2000 \s-1XXXX\s0 Shift_JISX
+文字集合に \s-1JIS\s0 X 0213:2000 を用いた Shift_JIS。
 .IP "\fBShift_JIS\-2004\fR" 4
 .IX Item "Shift_JIS-2004"
-\&\s-1XXXXX\s0 \s-1JIS\s0 X 0213:2004 \s-1XXXX\s0 Shift_JISX
+文字集合に \s-1JIS\s0 X 0213:2004 を用いた Shift_JIS。
 .IP "\fB\s-1CP932\s0\fR" 4
 .IX Item "CP932"
-Micorosft Code Page 932X
-\&\fB\-x\fR \s-1XXXXXXXXXXXXX\s0
+Micorosft Code Page 932。
+\&\fB\-x\fR が暗黙のうちに指定される。
 .IP "\fB\s-1UTF\-8\s0 \s-1UTF\-8N\s0\fR" 4
 .IX Item "UTF-8 UTF-8N"
-\&\s-1BOM\s0 \s-1XXX\s0 \s-1UTF\-8X\s0
-\&\fB\-w, \-W\fR \s-1XXXX\s0
+\&\s-1BOM\s0 無しの UTF\-8。
+\&\fB\-w, \-W\fR と同じ。
 .IP "\fB\s-1UTF\-8\-BOM\s0\fR" 4
 .IX Item "UTF-8-BOM"
-\&\s-1BOM\s0 \s-1XXX\s0 \s-1UTF\-8X\s0\fB\-w8\fR \s-1XXX\s0 \fB\-W\fR \s-1XXXX\s0
+\&\s-1BOM\s0 付きの UTF\-8。\fB\-w8\fR または \fB\-W\fR と同じ。
 .IP "\fB\s-1UTF8\-MAC\s0\fR" 4
 .IX Item "UTF8-MAC"
-\&\s-1UTF8\-MACXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXX\s0
+UTF8\-MAC。互換分解されたひらがな・カタカナ等を結合します。
+入力のみの対応です。
 .IP "\fB\s-1UTF\-16\s0 \s-1UTF\-16BE\-BOM\s0\fR" 4
 .IX Item "UTF-16 UTF-16BE-BOM"
-\&\s-1BOM\s0 \s-1XXX\s0 Big Endian X \s-1UTF\-16X\s0
-\&\fB\-w16B, \-W16B\fR \s-1XXXX\s0
+\&\s-1BOM\s0 有りで Big Endian の UTF\-16。
+\&\fB\-w16B, \-W16B\fR と同じ。
 .IP "\fB\s-1UTF\-16BE\s0\fR" 4
 .IX Item "UTF-16BE"
-\&\s-1BOM\s0 \s-1XXX\s0 Big Endian X \s-1UTF\-16X\s0
-\&\fB\-w16B0. \-W16B\fR \s-1XXXX\s0
+\&\s-1BOM\s0 無しで Big Endian の UTF\-16。
+\&\fB\-w16B0. \-W16B\fR と同じ。
 .IP "\fB\s-1UTF\-16LE\-BOM\s0\fR" 4
 .IX Item "UTF-16LE-BOM"
-\&\s-1BOM\s0 \s-1XXX\s0 Little Endian X \s-1UTF\-16X\s0
-\&\fB\-w16L, \-W16L\fR \s-1XXXX\s0
+\&\s-1BOM\s0 有りで Little Endian の UTF\-16。
+\&\fB\-w16L, \-W16L\fR と同じ。
 .IP "\fB\s-1UTF\-16LE\s0\fR" 4
 .IX Item "UTF-16LE"
-\&\s-1BOM\s0 \s-1XXX\s0 Little Endian X \s-1UTF\-16X\s0
-\&\fB\-w16L0, \-W16L\fR \s-1XXXX\s0
+\&\s-1BOM\s0 無しで Little Endian の UTF\-16。
+\&\fB\-w16L0, \-W16L\fR と同じ。
 .RE
 .RS 4
 .RE
 .IP "\fB\-\-fj \-\-unix \-\-mac \-\-msdos  \-\-windows\fR" 4
 .IX Item "--fj --unix --mac --msdos  --windows"
-\&\s-1XXXXXXXXXXXXXXXXXXX\s0
+これらのシステムに適した変換をします。
 .IP "\fB\-\-jis \-\-euc \-\-sjis \-\-mime \-\-base64\fR" 4
 .IX Item "--jis --euc --sjis --mime --base64"
-\&\s-1XXXXXXXXXXX\s0
+対応する変換をします。
 .IP "\fB\-\-hiragana \-\-katakana\fR" 4
 .IX Item "--hiragana --katakana"
-\&\s-1XXXXXXXXX\s0
+平仮名、片仮名変換
 .IP "\fB\-\-fb\-{skip, html, xml, perl, java, subchar}\fR" 4
 .IX Item "--fb-{skip, html, xml, perl, java, subchar}"
-Unicode \s-1XX\s0 Shift_JIS, EUC-JP, \s-1ISO\-2022\-JP\s0 \s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
+Unicode から Shift_JIS, EUC-JP, \s-1ISO\-2022\-JP\s0 に変換する際に、変換できなかった文字をどう扱うかを指定できます。
 .IP "\fB\-\-prefix=\f(BIescape character\fB\f(BItarget character\fB..\fR" 4
 .IX Item "--prefix=escape charactertarget character.."
-EUC-JP \s-1XX\s0 Shift_JIS \s-1XXXXXXX2\s0 \s-1XXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXX\s0 1 \s-1XXXXXXXXXXXX2\s0 \s-1XXXXXXXXXXXXXXXXXXXXXXXXX\s0
+EUC-JP から Shift_JIS への変換の際、2 バイト目
+に現れた文字の前にエスケープ文字をつけることができます。
+引数の 1 文字目がエスケープ文字、2 文字目以降にエスケープされるべき文字を指定します。
 .Sp
-\&\s-1XXXX\s0
+例えば、
 .Sp
-\&\-\-prefix=\e$@ XXXXXShift_JIS X 2 \s-1XXXX\s0 $ X @ \s-1XXXXXXXXX\s0 \e \s-1XXXXXXX\s0
-\&\-\-prefix=@@ XXXXXShift_JIS X 2 \s-1XXXX\s0 @ \s-1XXXXXXXXX\s0 @ \s-1XXXXXXX\s0
+\&\-\-prefix=\e$@ とすると、Shift_JIS の 2 文字目に $ か @ が来たら、その前に \e が挿入されます
+\&\-\-prefix=@@ とすると、Shift_JIS の 2 文字目に @ が来たら、その前に @ が挿入されます
 .IP "\fB\-\-no\-cp932ext\fR" 4
 .IX Item "--no-cp932ext"
-\&\s-1CP932\s0 \s-1XXXXXXXXXXNEC\s0 \s-1XXXXXNEC\s0 \s-1XX\s0 \s-1IBM\s0 \s-1XXXX\s0 (89\-92 X)XIBM \s-1XXXXXXXXXXXX\s0
+\&\s-1CP932\s0 において拡張された、NEC 特殊文字、NEC 選定 \s-1IBM\s0 拡張文字 (89\-92 区)、IBM 拡張文字を変換しません。
 .IP "\fB\-\-no\-best\-fit\-chars\fR" 4
 .IX Item "--no-best-fit-chars"
-Unicode \s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-Unicode \s-1XX\s0 Unicode \s-1XXXXXX\s0 \-x XXXXXXXXXnkf X \s-1UTF\s0 \s-1XXXXXXXXXXXXXXXXX\s0
-(\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0)
+Unicode からの変換の際に、往復安全性が確保されない文字の変換を行いません。
+Unicode から Unicode の変換の際に \-x と共に指定すると、nkf を \s-1UTF\s0 形式の変換に用いることができます。
+(逆に言えば、これを指定しないと一部の文字が保存されません)
 .Sp
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
+パスに関わる文字列を変換する際には、このオプションを指定することを強く推奨します。
 .IP "\fB\-\-cap\-input, \-\-url\-input\fR" 4
 .IX Item "--cap-input, --url-input"
-\&\s-1XXXX\s0 :X% \s-1XXX\s0 16 \s-1XXXXXXXXXX\s0
+それぞれ :、% に続く 16 進数を文字に変換する
 .IP "\fB\-\-numchar\-input\fR" 4
 .IX Item "--numchar-input"
-&#....; \s-1XXXX\s0 Unicode \s-1XXXXXXXXX\s0
+&#....; のような Unicode 文字参照を変換する
 .IP "\fB\-\-in\-place[=\f(BI\s-1SUFFIX\s0\fB]  \-\-overwrite[=\f(BI\s-1SUFFIX\s0\fB]\fR" 4
 .IX Item "--in-place[=SUFFIX]  --overwrite[=SUFFIX]"
-\&\s-1XXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXX\s0 inode \s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
+元のファイルを変換結果で置き換える。
+複数のファイルを書き換えることも可能。
+元のファイルのタイムスタンプとパーミッションが保持される。
+現在、作成日時や inode は変更されるが、将来にわたってこの実装のままである保証は無い。
 .IP "\fB\-\-guess\fR" 4
 .IX Item "--guess"
-\&\s-1XXXXXXXXXXXX\s0
+自動判別の結果を出力する
 .IP "\fB\-\-help\fR" 4
 .IX Item "--help"
-\&\s-1XXXXXXXXXXXXXXXX\s0
+コマンドの簡単な説明を表示する。
 .IP "\fB\-V\fR" 4
 .IX Item "-V"
-nkf \s-1XXXXXXXXX\s0
+nkf の設定を表示する。
 .IP "\fB\-v\fR \fB\-\-version\fR" 4
 .IX Item "-v --version"
-nkf \s-1XXXXXXXXXXXX\s0
+nkf のバージョンを表示する。
 .IP "\fB\-\-\fR" 4
 .IX Item "--"
-\&\s-1XXXXXXXXXXXXXXX\s0
+これ以降のオプションを無視する
 .IP "\fB\-\-exec\-in\fR" 4
 .IX Item "--exec-in"
-nkf [options] \-\-exec\-in cmd args... XXXXXcmd \s-1XXXX\s0 nkf \s-1XXXXXX\s0
-(config.h X \s-1EXEC_IO\s0 X define \s-1XXXXXXXXXXXXXX\s0)
+nkf [options] \-\-exec\-in cmd args... とやると、cmd の出力を nkf の入力とする
+(config.h で \s-1EXEC_IO\s0 を define してコンパイルした時のみ有効)
 .IP "\fB\-\-exec\-out\fR" 4
 .IX Item "--exec-out"
-nkf [options] \-\-exec\-out cmd args... XXXXXnkf \s-1XXXX\s0 cmd \s-1XXXXXX\s0
-(config.h X \s-1EXEC_IO\s0 X define \s-1XXXXXXXXXXXXXX\s0)
+nkf [options] \-\-exec\-out cmd args... とやると、nkf の出力を cmd の入力とする
+(config.h で \s-1EXEC_IO\s0 を define してコンパイルした時のみ有効)
 .SH "BUGS"
 .IX Header "BUGS"
-XXXXXXXXXXXXXXXShift_JIS X \s-1EUC\s0 \s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\s0
-\&\s-1XXXXX7\s0 \s-1XXX\s0 \s-1JIS\s0 \s-1XXXXXXXXX\s0
-Shift_JISXEUC \s-1XXXXXXXXXXXXXXXXX\s0
-XXXXShift_JIS \s-1XXXXXXXXX\s0 (\s-1JIS\s0 X 0201 \s-1XXX\s0) X \s-1EUC\s0 \s-1XXXXXXXXXX\s0
-\&\s-1XXXXXXXXXXXXX\s0 Shift_JIS \s-1XXXXXXXXXXXXXXXXXXXX\s0
-(XXXXShift_JIS X \s-1EUC\s0 \s-1XXXXXX\s0 \s-1EUC\s0 \s-1XXXX\s0)
+実は、入力のコード系の判定は、Shift_JIS と \s-1EUC\s0 との自動判定であり、最初に特定できる文字が来た時点で確定してしまう。
+そして、特定不能の間は保留バッファにためておかれ、確定後に処理される。
+このため、7 ビット \s-1JIS\s0 は常に認識される。
+Shift_JIS、EUC 混合もほとんどの場合は問題がない。
+ただし、Shift_JIS のいわゆる半角カナ (\s-1JIS\s0 X 0201 片仮名) と \s-1EUC\s0 漢字が識別できない。
+したがって、デフォルトでは Shift_JIS のいわゆる半角カナはないと仮定している。
+(つまり、Shift_JIS か \s-1EUC\s0 か迷った時は \s-1EUC\s0 とみなす)
 .PP
-Unicode \s-1XXXXXXXXXXXXXXXXXXXXXXX\s0\fB\-\-no\-best\-fit\-chars\fR \s-1XXXXXXXUTF\-8\s0 X \s-1UTF\-16\s0 \s-1XXXXXXXXXXXXXXXXX\s0
+Unicode の日本語以外の部分の変換には対応していないが、\fB\-\-no\-best\-fit\-chars\fR を指定すれば、UTF\-8 と \s-1UTF\-16\s0 の相互変換をすることは可能である。
 .SH "ACKNOWLEDGE"
 .IX Header "ACKNOWLEDGE"
-\&\s-1XXXXXXXXXXXXXXXXXXXXXXX\s0
+以下のかたがた、どうもありがとうございました。
 .SS "nkf 1.4"
 .IX Subsection "nkf 1.4"
 morb@fujitsu, kiss@ayumi.stars.flab.fujitsu, cen122@flab.fujitsu,
-yuki@flab.fujitsu XXfujitsu & flab.fujitsu \s-1XXXXXXXXXXX\s0
+yuki@flab.fujitsu 他、fujitsu & flab.fujitsu の皆さんの協力に感謝。
 shinoda@cs.titech, kato@cs.titech, uematsu@cs.titech  \s-1TNX\s0
 kono@ie.u\-ryukyu.ac.jp
 void@merope.pleiades.or.jp (Kusakabe Youichi)
@@ -566,8 +504,8 @@ kono@ie.u\-ryukyu.ac.jp (Shinji Kono)
 GHG00637@nifty\-serve.or.jp (\s-1COW\s0)
 .SS "nkf 1.9\-"
 .IX Subsection "nkf 1.9-"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
+.IP "河野真治" 4
+.IX Item "河野真治"
 .PD 0
 .IP "Akio Furukawa" 4
 .IX Item "Akio Furukawa"
@@ -579,8 +517,8 @@ GHG00637@nifty\-serve.or.jp (\s-1COW\s0)
 .IX Item "Fumitaka Kitagawa"
 .IP "Hiroaki Sengoku" 4
 .IX Item "Hiroaki Sengoku"
-.IP "Ikuhiro \s-1MORITA\s0 (\s-1XX\s0 \s-1XX\s0)" 4
-.IX Item "Ikuhiro MORITA (XX XX)"
+.IP "Ikuhiro \s-1MORITA\s0 (森田 育宏)" 4
+.IX Item "Ikuhiro MORITA (森田 育宏)"
 .IP "Yoshiharu \s-1ITO\s0" 4
 .IX Item "Yoshiharu ITO"
 .IP "Junn Ohta" 4
@@ -599,10 +537,10 @@ GHG00637@nifty\-serve.or.jp (\s-1COW\s0)
 .IX Item "Kiwamu Aoyama"
 .IP "Koichi Hirayama" 4
 .IX Item "Koichi Hirayama"
-.IP "Mitsuru Hase (\s-1XXXX\s0)" 4
-.IX Item "Mitsuru Hase (XXXX)"
-.IP "\s-1OHARA\s0 Shigeki (\s-1XX\s0 \s-1XX\s0)" 4
-.IX Item "OHARA Shigeki (XX XX)"
+.IP "Mitsuru Hase (長谷　満)" 4
+.IX Item "Mitsuru Hase (長谷　満)"
+.IP "\s-1OHARA\s0 Shigeki (大原 重樹)" 4
+.IX Item "OHARA Shigeki (大原 重樹)"
 .IP "Rei \s-1FURUKAWA\s0" 4
 .IX Item "Rei FURUKAWA"
 .IP "Satoru Takabayashi" 4
@@ -619,104 +557,104 @@ GHG00637@nifty\-serve.or.jp (\s-1COW\s0)
 .IX Item "TSUCHIYA Masatoshi"
 .IP "Tsutomu Sakai" 4
 .IX Item "Tsutomu Sakai"
-.IP "\s-1YAMASHITA\s0 Junji (\s-1XX\s0 \s-1XX\s0)" 4
-.IX Item "YAMASHITA Junji (XX XX)"
+.IP "\s-1YAMASHITA\s0 Junji (山下 純司)" 4
+.IX Item "YAMASHITA Junji (山下 純司)"
 .IP "Yasuyuki Sato" 4
 .IX Item "Yasuyuki Sato"
 .IP "Yoshiaki Yanagihara" 4
 .IX Item "Yoshiaki Yanagihara"
 .IP "hat@so\-net" 4
 .IX Item "hat@so-net"
-.IP "\s-1XXXXXXXX\s0" 4
-.IX Item "XXXXXXXX"
-.IP "\s-1XXX\s0 / \s-1XX\s0 \s-1XX\s0" 4
-.IX Item "XXX / XX XX"
-.IP "XXdais" 4
-.IX Item "XXdais"
-.IP "\s-1XXX\s0 \s-1XXXX\s0" 4
-.IX Item "XXX XXXX"
-.IP "\s-1XXXXXXXXXXX\s0(X)XX" 4
-.IX Item "XXXXXXXXXXX(X)XX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XX\s0 \s-1XX\s0(\s-1SHIOZAKI\s0 Takehiko)" 4
-.IX Item "XX XX(SHIOZAKI Takehiko)"
-.IP "\s-1XXXX\s0 (Media Lab.)" 4
-.IX Item "XXXX (Media Lab.)"
-.IP "\s-1XX\s0 \s-1XX\s0" 4
-.IX Item "XX XX"
-.IP "\s-1XXXXXXXXXXXX\s0" 4
-.IX Item "XXXXXXXXXXXX"
-.IP "\s-1XX\s0 \s-1XX\s0" 4
-.IX Item "XX XX"
-.IP "\s-1XX\s0 \s-1XX\s0 (\s-1XXXX\s0 \s-1XXX\s0)" 4
-.IX Item "XX XX (XXXX XXX)"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XX\s0" 4
-.IX Item "XX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XX\s0 X" 4
-.IX Item "XX X"
-.IP "\s-1XX\s0 \s-1XX\s0" 4
-.IX Item "XX XX"
-.IP "\s-1XXXX\s0 (\s-1XXXXXX\s0)" 4
-.IX Item "XXXX (XXXXXX)"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XX\s0 \s-1XX\s0 (Koji Arai)" 4
-.IX Item "XX XX (Koji Arai)"
-.IP "\s-1XXXXX\s0" 4
-.IX Item "XXXXX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XX\s0 \s-1XX\s0" 4
-.IX Item "XX XX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XXXXXXXXX\s0" 4
-.IX Item "XXXXXXXXX"
-.IP "\s-1XX\s0 \s-1XXX\s0 == Eiichiro Itani" 4
-.IX Item "XX XXX == Eiichiro Itani"
-.IP "\s-1XXXXX\s0" 4
-.IX Item "XXXXX"
-.IP "\s-1XX\s0 \s-1XX\s0 (Masayuki Hatta)" 4
-.IX Item "XX XX (Masayuki Hatta)"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XX\s0 \s-1XX\s0" 4
-.IX Item "XX XX"
-.IP "\s-1XX\s0 (\s-1XXXXXXXXXX\s0)" 4
-.IX Item "XX (XXXXXXXXXX)"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XXXXXXX\s0" 4
-.IX Item "XXXXXXX"
-.IP "\s-1XX\s0 \s-1XX\s0" 4
-.IX Item "XX XX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "X  X" 4
-.IX Item "X  X"
-.IP "\s-1XX\s0 X" 4
-.IX Item "XX X"
-.IP "\s-1XX\s0 \s-1XX\s0" 4
-.IX Item "XX XX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XXXX\s0" 4
-.IX Item "XXXX"
-.IP "\s-1XXXXXXXX\s0" 4
-.IX Item "XXXXXXXX"
+.IP "ＤＣＣ技術部渡辺" 4
+.IX Item "ＤＣＣ技術部渡辺"
+.IP "かとぺ / 加藤 貴司" 4
+.IX Item "かとぺ / 加藤 貴司"
+.IP "かべdais" 4
+.IX Item "かべdais"
+.IP "ひろせ まさあき" 4
+.IX Item "ひろせ まさあき"
+.IP "イントラネットシステム(株)松尾" 4
+.IX Item "イントラネットシステム(株)松尾"
+.IP "鵜飼文敏" 4
+.IX Item "鵜飼文敏"
+.IP "塩崎 毅彦(\s-1SHIOZAKI\s0 Takehiko)" 4
+.IX Item "塩崎 毅彦(SHIOZAKI Takehiko)"
+.IP "河村雅夫 (Media Lab.)" 4
+.IX Item "河村雅夫 (Media Lab.)"
+.IP "河野 康司" 4
+.IX Item "河野 康司"
+.IP "喜瀬“冬猫”浩＠南国沖縄" 4
+.IX Item "喜瀬“冬猫”浩＠南国沖縄"
+.IP "金井 智彦" 4
+.IX Item "金井 智彦"
+.IP "桂田 祐史 (かつらだ まさし)" 4
+.IX Item "桂田 祐史 (かつらだ まさし)"
+.IP "高橋宜盟" 4
+.IX Item "高橋宜盟"
+.IP "国吉" 4
+.IX Item "国吉"
+.IP "黒江明彦" 4
+.IX Item "黒江明彦"
+.IP "三宅正泰" 4
+.IX Item "三宅正泰"
+.IP "山倉 真" 4
+.IX Item "山倉 真"
+.IP "山本 芳人" 4
+.IX Item "山本 芳人"
+.IP "山野裕司 (やまのゆうじ)" 4
+.IX Item "山野裕司 (やまのゆうじ)"
+.IP "重村法克" 4
+.IX Item "重村法克"
+.IP "小島基靖" 4
+.IX Item "小島基靖"
+.IP "上田　健" 4
+.IX Item "上田　健"
+.IP "新井 康司 (Koji Arai)" 4
+.IX Item "新井 康司 (Koji Arai)"
+.IP "森　千絵子" 4
+.IX Item "森　千絵子"
+.IP "森　和彦" 4
+.IX Item "森　和彦"
+.IP "水野 貴文" 4
+.IX Item "水野 貴文"
+.IP "前地和俊" 4
+.IX Item "前地和俊"
+.IP "仲宗根＠並列信頼研" 4
+.IX Item "仲宗根＠並列信頼研"
+.IP "猪谷 英一郎 == Eiichiro Itani" 4
+.IX Item "猪谷 英一郎 == Eiichiro Itani"
+.IP "藤原　秀行" 4
+.IX Item "藤原　秀行"
+.IP "八田 真行 (Masayuki Hatta)" 4
+.IX Item "八田 真行 (Masayuki Hatta)"
+.IP "尾川敏也" 4
+.IX Item "尾川敏也"
+.IP "望月 孝志" 4
+.IX Item "望月 孝志"
+.IP "木村 (＠筑波大学数学研究科)" 4
+.IX Item "木村 (＠筑波大学数学研究科)"
+.IP "野村俊彰" 4
+.IX Item "野村俊彰"
+.IP "有岡＠ミノルタ" 4
+.IX Item "有岡＠ミノルタ"
+.IP "有沢 明宏" 4
+.IX Item "有沢 明宏"
+.IP "落合一喜" 4
+.IX Item "落合一喜"
+.IP "林  亮" 4
+.IX Item "林  亮"
+.IP "國安 治" 4
+.IX Item "國安 治"
+.IP "木下 雄介" 4
+.IX Item "木下 雄介"
+.IP "田中隆裕" 4
+.IX Item "田中隆裕"
+.IP "鶴谷直樹" 4
+.IX Item "鶴谷直樹"
+.IP "中田伸悦" 4
+.IX Item "中田伸悦"
+.IP "わたなべひろふみ" 4
+.IX Item "わたなべひろふみ"
 .IP "MoonWolf" 4
 .IX Item "MoonWolf"
 .IP "sava" 4
@@ -724,15 +662,15 @@ GHG00637@nifty\-serve.or.jp (\s-1COW\s0)
 .PD
 .SH "AUTHOR"
 .IX Header "AUTHOR"
-.IP "\s-1XX\s0  X        ichikawa@flab.fujitsu.co.jp (was ichikawa@fujitsu.JUNET)" 4
-.IX Item "XX  X        ichikawa@flab.fujitsu.co.jp (was ichikawa@fujitsu.JUNET)"
+.IP "市川  至        ichikawa@flab.fujitsu.co.jp (was ichikawa@fujitsu.JUNET)" 4
+.IX Item "市川  至        ichikawa@flab.fujitsu.co.jp (was ichikawa@fujitsu.JUNET)"
 .PD 0
-.IP "\s-1XX\s0  \s-1XX\s0      kono@ie.u\-ryukyu.acjp" 4
-.IX Item "XX  XX      kono@ie.u-ryukyu.acjp"
+.IP "河野  真治      kono@ie.u\-ryukyu.acjp" 4
+.IX Item "河野  真治      kono@ie.u-ryukyu.acjp"
 .IP "Rei \s-1FURUKAWA\s0    furukawa@tcp\-ip.or.jp" 4
 .IX Item "Rei FURUKAWA    furukawa@tcp-ip.or.jp"
-.IP "\s-1XX\s0" 4
-.IX Item "XX"
+.IP "成瀬" 4
+.IX Item "成瀬"
 .IP "mastodon" 4
 .IX Item "mastodon"
 .PD
